manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -163,6 +163,7 @@ data:
   "max-pvscsi-targets-per-vm": "true"
   "multi-vcenter-csi-topology": "false"
   "csi-internal-generated-cluster-id": "false"
+  "improved-csi-tasks-listview": "false"
```

Comment:
feature name should be `listview-tasks`. No need to add improve prefix.
We added an improved prefix to the topology feature, as the topology feature was already there in the driver and we did changes in the logic.

Reply:
Fixed.